packages/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.1
```

Comment:
nit: This has a merge conflict on it now.

Reply:
Resolved conflict!